src/sector.jl

```diff
@@ -1,55 +1,64 @@
-export SUNIrrep
-
 # is this type piracy?
 const SU₃ = SU{3}
 const SU₄ = SU{4}
 const SU₅ = SU{5}
-TensorKit.type_repr(::Type{SU₃}) = "SU₃"
-TensorKit.type_repr(::Type{SU₄}) = "SU₄"
-TensorKit.type_repr(::Type{SU₅}) = "SU₅"
-Base.getindex(::TensorKit.IrrepTable, ::Type{SU{N}}) where {N} = SUNIrrep{N}
+TensorKitSectors.type_repr(::Type{SU₃}) = "SU₃"
+TensorKitSectors.type_repr(::Type{SU₄}) = "SU₄"
+TensorKitSectors.type_repr(::Type{SU₅}) = "SU₅"
+Base.getindex(::TensorKitSectors.IrrepTable, ::Type{SU{N}}) where {N} = SUNIrrep{N}
```

Comment:
Type piracy! 😄

Reply:
Oh, I did not see the comment above. So yes, it is. But that's why it is good that it is only a gentlemen's agreement and not an enforced rule.